src/native/third_party/isa-l.gyp

```diff
@@ -313,11 +313,13 @@
                 'isa-l_crypto/include/',
                 'isa-l_crypto/sha256_mb/',
             ],
+            'sources': [
+                'isa-l_crypto/sha256_mb/sha256_ctx_base.c',
+            ],
```

Comment:
same as for sha1 - isn't there an aliases file for sha256 too that should be added to common sources?